gossip/src/cluster_info_metrics.rs

```diff
@@ -138,6 +142,7 @@ pub struct GossipStats {
     pub(crate) packets_sent_pull_responses_count: Counter,
     pub(crate) packets_sent_push_messages_count: Counter,
     pub(crate) process_gossip_packets_time: Counter,
+    pub(crate) process_gossip_packets_itrs_since_last_report: Counter,
```

Comment:
can you please keep the fields in this struct alphabetically sorted ?
that makes it easier to resolve merge conflicts or skim through the list of metrics.